bootstrap.sh

```diff
@@ -36,8 +36,9 @@ all_module_files=`find openfisca_country_template -type f`
 set -x
 
 # Use intermediate backup files (`-i`) with a weird syntax due to lack of portable 'no backup' option. See https://stackoverflow.com/q/5694228/594053.
-sed -i.template "s|country_template|$lowercase_country_name|g" README.md setup.py .circleci/config.yml Makefile MANIFEST.in $all_module_files
-sed -i.template "s|Country-Template|$COUNTRY_NAME|g" README.md setup.py .github/PULL_REQUEST_TEMPLATE.md CONTRIBUTING.md $all_module_files
+# If you're using MacOS, add LC_ALL=C before sed https://stackoverflow.com/questions/19242275/re-error-illegal-byte-sequence-on-mac-os-x/19770395#19770395
```

Comment:
I don't have any warning or issue with macOS 11.6. What version did trigger the addition of this comment? 🙂
In any case, the aim with this bootstrapping script is to be fully automatic. If we need adjustments, they should be automated, we cannot expect users to edit the script itself.